spacy_classifiers.ipynb

```diff
@@ -3946,27 +3921,9 @@
    "cell_type": "code",
    "execution_count": 12,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Collecting msgpack-numpy<0.4.4.0\n",
-      "  Downloading https://files.pythonhosted.org/packages/ad/45/464be6da85b5ca893cfcbd5de3b31a6710f636ccb8521b17bd4110a08d94/msgpack_numpy-0.4.3.2-py2.py3-none-any.whl\n",
-      "Requirement already satisfied: msgpack>=0.3.0 in /srv/conda/lib/python3.6/site-packages (from msgpack-numpy<0.4.4.0) (0.6.1)\n",
-      "Requirement already satisfied: numpy>=1.9.0 in /srv/conda/lib/python3.6/site-packages (from msgpack-numpy<0.4.4.0) (1.14.3)\n",
-      "Installing collected packages: msgpack-numpy\n",
-      "  Found existing installation: msgpack-numpy 0.4.4.2\n",
-      "    Uninstalling msgpack-numpy-0.4.4.2:\n",
-      "      Successfully uninstalled msgpack-numpy-0.4.4.2\n",
-      "Successfully installed msgpack-numpy-0.4.3.2\n",
-      "\u001b[33mYou are using pip version 18.1, however version 19.0.3 is available.\n",
-      "You should consider upgrading via the 'pip install --upgrade pip' command.\u001b[0m\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "!pip install \"msgpack-numpy<0.4.4.0\""
+    "# !pip install \"msgpack-numpy<0.4.4.0\"  # tjm removed, not needed?"
```

Comment:
This is required for incompatible conda versions. It doesn't throw an error in your machine probably because the right version is downloaded as part of an earlier project. Let this line remain uncommented. We will make the required changes during conversion to Python,

Reply:
AOK